src/screens/Dashboard/DeparturesWidget.tsx

```diff
@@ -0,0 +1,115 @@
+import {getFavouriteDepartures} from '@atb/api/departures';
+import {StopPlaceGroup} from '@atb/api/departures/types';
+import ThemeText from '@atb/components/text';
+import QuaySection from '@atb/departure-list/section-items/quay-section';
+import {useFavorites} from '@atb/favorites';
+import {useGeolocationState} from '@atb/GeolocationContext';
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {StyleSheet} from '@atb/theme';
+import {useTranslation} from '@atb/translations';
+import DeparturesTexts from '@atb/translations/screens/Departures';
+import {useIsFocused, useNavigation} from '@react-navigation/native';
+
+import React, {useEffect, useState} from 'react';
+import {View} from 'react-native';
+
+const FavouritesWidget: React.FC = () => {
+  const styles = useStyles();
+  const {t} = useTranslation();
+  const {favoriteDepartures} = useFavorites();
+  const {location} = useGeolocationState();
+  const [polling, setPolling] = useState(false);
+  const [favResults, setFavResults] = useState<StopPlaceGroup[]>([]);
+  const [searchDate, setSearchDate] = useState<string>('');
+  const navigation = useNavigation();
+  const isFocused = useIsFocused();
+  const {favourite_departures_poll_interval} = useRemoteConfig();
+
+  const fetchFavouriteDepartures = async () => {
+    await getFavouriteDepartures(favoriteDepartures).then((data) => {
+      if (data) {
+        setFavResults(data);
+        setSearchDate(new Date().toISOString());
+      }
+    });
+  };
+
+  // timer orchestration
+  useEffect(() => {
+    let interval: NodeJS.Timeout | undefined = undefined;
+    if (polling) {
+      fetchFavouriteDepartures();
+      interval = setInterval(
+        fetchFavouriteDepartures,
+        favourite_departures_poll_interval,
+      );
+    } else {
+      if (interval) {
+        clearInterval(interval);
+      }
+    }
+
+    return () => {
+      if (interval) {
+        clearInterval(interval);
+      }
+    };
+  }, [polling]);
+
+  // do polling only when screen has focus and user has faourites.
```

Comment:
If we're unsure about whether to use "favorite" or "favourite", we can always settle with "faourite" 😆

```suggestion
  // do polling only when screen has focus and user has favourites.
```

Reply:
:D fixed